flink/json-taskmanagers.yml

```diff
@@ -0,0 +1,120 @@
+type: json
+data:
+  url: /v1/taskmanagers
+  server: ${FLINK_HOSTPORT}
+
+observation:
+  - name: taskmanagers
+    metricNamespace: flink
+    path: $.taskmanagers[?(@.id=${taskmanagerid})]
+
+    metric:
+      - name: taskmanagers.cores
+        source: json:/v1/taskmanagers/${taskmanagerid} $.hardware.cpuCores
+        type: long_gauge
+        label: cpu cores
+        description: cpu cores
+
+      - name: taskmanagers.memory.total
+        source: json:/v1/taskmanagers/${taskmanagerid} $.hardware.physicalMemory
+        type: long_gauge
+        unit: bytes
+        label: total memory
+        description: physical memory
+
+      - name: taskmanagers.memory.managed
+        source: json:/v1/taskmanagers/${taskmanagerid} $.hardware.managedMemory
+        type: long_gauge
+        unit: bytes
+        label: managed memory
+        description: managed memory
+
+      - name: taskmanagers.heap.used
```

Comment:
Insert `memory` after `taskmanagers`, same for other metrics.

Reply:
DONE